src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/step/snapshot/AttemptSnapshotStep.kt

```diff
@@ -122,15 +143,16 @@ class AttemptSnapshotStep(
     override fun getUpdatedManagedIndexMetaData(currentMetaData: ManagedIndexMetaData): ManagedIndexMetaData {
         val currentActionMetaData = currentMetaData.actionMetaData
         return currentMetaData.copy(
-                actionMetaData = currentActionMetaData?.copy(actionProperties = ActionProperties(snapshotName = snapshotName)),
-                stepMetaData = StepMetaData(name, getStepStartTime().toEpochMilli(), stepStatus),
-                transitionTo = null,
-                info = info
+            actionMetaData = currentActionMetaData?.copy(actionProperties = ActionProperties(snapshotName = snapshotName)),
+            stepMetaData = StepMetaData(name, getStepStartTime().toEpochMilli(), stepStatus),
+            transitionTo = null,
+            info = info
         )
     }
 
     companion object {
         const val name = "attempt_snapshot"
+        fun getBlockedMessage(denyList: List<String>, repoName: String) = "snapshot repository [$repoName] is blocked in $denyList"
```

Comment:
Fix consistency with other messages. Recommend changing to `Snapshot repository [$repoName] is blocked in $denyList [index=$index]`

Reply:
addressed